doc/how_to/waveform_extractor_to_sorting_analyzer.rst

```diff
@@ -0,0 +1,572 @@
+=========================================
+From WaveformExtractor to SortingAnalyzer
+=========================================
+
+From spikeinterface version 0.101, the internal structure of the postprocessing module has
+changed. We explain the motivation for the change in more detail `below <#why-change>`_.
+
+From the user point of view, the key change is the deletion of the ``WaveformExtractor`` class and the addition of
+the new ``SortingAnalyzer`` class. Hence any code using `WaveformExtractors` will have to be updated.
+If you want to continue using ``WaveformExtractor`` you need to use spikeinterface versions 0.100 and
+below.
+
+Updating your old code should be straightforward. On this page, we demonstrate how to `convert old WaveformExtractor folders
+to new SortingAnalyzer folders <#convert-a-waveformextractor-folder-to-a-sortinganalyzer-folder>`_ and have written a
+`code dictionary <#dictionary-between-sortinganalyzer-and-waveformextractor>`_ which should make updating your codebase simple.
+
+Why change?
+^^^^^^^^^^^
+
+Previously, a lot of post-processing was handled by the ``WaveformExtractor`` class. However, as the name suggests, this singles out waveforms as special. But many outputs don't care about waveforms. And this isn't just a naming problem: you had to create a WaveformExtractor object (and thus calculate waveforms) to calculate e.g. spike_amplitudes, which didn't necessarily depend on them. Then to access this data you had to access the WaveformExtractor. We felt that this was confusing to the user: why are the spike amplitudes stored in a waveform class?? This mixing of waveforms with non-waveform objects also makes the codebase messy and harder to develop.
```

Comment:
spike_amplitudes does require knowledge of the waveforms :) but to get around this we have fast_templates which still requires using the random_spikes to get the info without officially calculating the waveforms. Maybe we pick something that is not an edge case and use correlograms as an example here since that really only requires the spike times?